charts/scalardb-cluster/README.md

```diff
@@ -65,4 +65,9 @@ Current chart version is `2.0.0-SNAPSHOT`
 | scalardbCluster.strategy.rollingUpdate.maxSurge | string | `"25%"` | The number of pods that can be created above the desired amount of pods during an update |
 | scalardbCluster.strategy.rollingUpdate.maxUnavailable | string | `"25%"` | The number of pods that can be unavailable during the update process |
 | scalardbCluster.strategy.type | string | `"RollingUpdate"` | New pods are added gradually, and old pods are terminated gradually, e.g: Recreate or RollingUpdate |
+| scalardbCluster.tls.caRootCertSecret | string | `""` | Secret name that includes the custom CA root certificate for TLS communication. |
```

Comment:
> Secret name that includes the custom CA root certificate

Sorry, the `includes` seems unclear to me. Is it a common expression in k8s's Secret?

Reply:
My previous comment was lack of information, I think. What I meant was the description sounded the Secret Name itself might be the cert content, although it was a silly idea. So, the following would be clearer?

```
Name of the Secret containing the custom CA root certificate for TLS communication
```

Reply:
Thank you for your review!
I agree with you. The current description is a bit unclear. 
I fixed it in efe4fe3fb82de91e1fb0cdbbbb498a1faf7fcb8b.